app/src/main/res/values/strings.xml

```diff
@@ -653,6 +653,8 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_barrier_bicycle_type_multiple">Chicane - more than two barriers</string>
     <string name="quest_barrier_bicycle_type_not_cycle_barrier">Not a cycle barrier, but some other barrier</string>
 
+    <string name="quest_bicycle_incline_title">Which direction leads upwards for this way?</string>
```

Comment:
Or just 

```suggestion
    <string name="quest_bicycle_incline_title">Which direction leads upwards here?</string>
```
? After all, the type of thing is already mentioned in the quest hint label. (Could maybe also delete the special string for the steps quests then)

Reply:
For steps it can be actually useful to strongly focus on steps, but here generic way is definitely pointless